docs/intro/quickstart/setup.rst

```diff
@@ -0,0 +1,69 @@
+.. _ref_quickstart_setup:
+
+===========================
+Setting up your environment
```

Comment:
We should also include `createClient` here as we're already jumping to using Gel in Next.js and this file seems to be a good opportunity to explain the Gel-related files + how to get a client.

(I think it was already mentioned somewhere, but I couldn't find it, sorry for repeating)